v03_pipeline/lib/reference_datasets/reference_dataset.py

```diff
@@ -92,36 +102,32 @@ def requires(self) -> ReferenceDataset:
 
 
 CONFIG = {
-    ReferenceDataset.cadd: {
+    ReferenceDataset.dbnsfp: {
+        ENUMS: {
+            'MutationTaster_pred': ['D', 'A', 'N', 'P'],
+        },
         ReferenceGenome.GRCh37: {
             DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
             VERSION: '1.0',
-            RAW_DATASET_PATH: [
-                'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh37/whole_genome_SNVs.tsv.gz',
-                'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh37/gnomad.genomes-exomes.r4.0.indel.tsv.gz',
-            ],
+            RAW_DATASET_PATH: 'https://dbnsfp.s3.amazonaws.com/dbNSFP4.7a.zip',
         },
         ReferenceGenome.GRCh38: {
-            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL]),
+            DATASET_TYPES: frozenset([DatasetType.SNV_INDEL, DatasetType.MITO]),
             VERSION: '1.0',
-            RAW_DATASET_PATH: [
-                'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh38/whole_genome_SNVs.tsv.gz',
-                'https://krishna.gs.washington.edu/download/CADD/v1.7/GRCh38/gnomad.genomes.r4.0.indel.tsv.gz',
-            ],
+            RAW_DATASET_PATH: 'https://dbnsfp.s3.amazonaws.com/dbNSFP4.7a.zip',
         },
     },
     ReferenceDataset.clinvar: {
+        ENUMS: clinvar.ENUMS,
```

Comment:
👍 this looks ok to me!  I know at some point in the past we've had different enums for the different reference genomes but we can handle that later on if need me.

Reply:
I think yes, that's a great idea.  There isn't a good reason to have them in the data module now if they are only referenced here.